### PR TITLE
Feature/#130341347/landing page

### PR DESCRIPTION
**Pull request by cyrielo**
#### What does this PR do?

This PR integrates a new landing page based on the mock-up created for the project.
#### Description of Task to be completed?
- Users should be able to see an overview of what the game is about.
- How it's played
- How it can be used to donate to charity
#### How should this be manually tested?

N/A
#### What are the relevant pivotal tracker stories?
#130341347 Users should see a landing page with effective UI/UX
#### Screenshots (if appropriate)

screenshot from 2016-10-05 14-50-05
screenshot from 2016-10-05 14-49-49

### CODE REVIEW

**Comment by andela-rsadiq** [comment]
@cyrielo fix conflict in `public/views/index.html`, so I can go on with merging!

**Review by ruqoyyasadiq — CHANGES_REQUESTED** [review]
@cyrielo fix conflicts in `public/views/index.html`, so I can move on to merge.

File: public/css/landing.css [hunk 1]
```diff
@@ -0,0 +1,290 @@
+@font-face {
```
Comment (ruqoyyasadiq) [inline public/css/landing.css:1]: are styles now written in `css`. My thought is that we are writing in `sass` and converting to `css` with the aid of our task runner (gulp).

**Review by ruqoyyasadiq — APPROVED** [review]
@cyrielo I would hate to hold this PR back because of the use of `css` as against `sass`. But note that all stylesheets manually created in `css` should be converted to `sass`. So should all newly created stylesheet should be written with sass.

**Pull request merged**